include/ibm/management_console_rest.hpp

```diff
@@ -944,8 +945,8 @@ inline void getCSREntryAck(const std::shared_ptr<bmcweb::AsyncResp>& asyncResp,
             return;
         }
 
-        BMCWEB_LOG_CRITICAL << "VMI Cert Entry " << entryId
-                            << " status property value " << *status;
+        BMCWEB_LOG_DEBUG << "VMI Cert Entry " << entryId
+                         << "new status property value " << *status;
```

Comment:
these traces are helpful in debugging error cases.
all these vmi Certificate traces were added for debugging error cases.
otherwise we don't have any info to debug failure case in production

Reply:
reverting back to critical for all changes in this file , but to show that its just information adding INFO: before string  .
BMCWEB_LOG_CRITICAL << "INFO:Message_string  "